### ISSUE TITLE
Suggestion: Playing Audio Files in XOWA

### ISSUE THREAD

**Issue by Ope30** [gnosygnu/xowa#166]
It is not possible to play Audio Files at the moment. Is there a reason behind this? However, I wouldn't mind seeing it being possible in the next build!

**Comment by gnosygnu** [comment]
Hey. The answer is similar to #164 . Downloading audio files would take a lot of space.

You should be able to download them manually by clicking on the play link in the article. If this doesn't work, let me know which article and link, and I'll take a look.

Thanks!

**Comment by Ope30** [comment]
> Hey. The answer is similar to #164 . Downloading audio files would take a lot of space.

True. I would delete the audio file anyway once I'm done hearing what's in. :d

> You should be able to download them manually by clicking on the play link in the article. If this doesn't work, let me know which article and link, and I'll take a look.

Unfortunately, that does not work for me! de.wikipedia.org/wiki/Zweiter Weltkrieg offers audio files to play with. None of them work for me. I click on the "Play" icon, but that just brings me to the top of the article. Also, literally every article that has audio files doesn't let me play them.

**Comment by gnosygnu** [comment]
Thanks for the example. I tried clicking on "Teil 1: Vorgeschichte" and can confirm it didn't play.

Let me look at it for this weekend
